tools/steamsync/steamsync.config.yaml

```diff
@@ -87,4 +87,8 @@ workshop:
   tactical_glide: 2682906097
   fire_support_plus: 2699465073
   grad_civs: 2140453587
-  crows_zeus: 2447965207
\ No newline at end of file
+  crows_zeus: 2447965207
+  dmp: 1990653162
```

Comment:
Give this an actual name so mission makers and players know what they're selecting

Reply:
doneski